tests/integration/test_ethereum_tester.py

```diff
@@ -298,6 +298,9 @@ def test_eth_chainId(self, web3):
         assert is_integer(chain_id)
         assert chain_id == 61
 
+    @pytest.mark.xfail(raises=KeyError, reason="ethereum tester doesn't return 'to' key")
+    def test_eth_getTransactionReceipt_unmined(self, eth_tester, web3, unlocked_account):
```

Comment:
I believe this should be `def test_eth_getTransactionReceipt_mined`. Sorry I mistyped earlier.